doc/toc.yml

```diff
@@ -69,12 +69,20 @@
           href: Overview/Mvux/Feeds.md
         - name: States
           href: Overview/Mvux/States.md
-        - name: How to create an MVUX project
-          href: Overview/Mvux/How-tos/MvuxProject.md
-        - name: How to create a Feed
-          href: Overview/Mvux/How-tos/SimpleFeed.md
-        - name: How to create a State
-          href: Overview/Mvux/How-tos/SimpleState.md
+        - name: Advanced topics
+          href: Overview/Mvux/Advanced.md
+        - name: How to
```

Comment:
"Tutorials" is used elsewhere
![image](https://user-images.githubusercontent.com/1614057/234908787-5a0cd5f7-85fa-495a-804b-ad22c558a645.png)

Reply:
Since all other PRs have now been merged, it's now a good time to rename this toc in one go. I'll update.

Reply:
Done